requirements.txt

```diff
@@ -0,0 +1,26 @@
+absl-py==0.8.0
+astor==0.8.0
+cloudpickle==1.2.1
+future==0.17.1
+gast==0.2.2
+google-pasta==0.1.7
+grpcio==1.23.0
+gym==0.14.0
+h5py==2.9.0
+Keras==2.2.4
+Keras-Applications==1.0.8
+Keras-Preprocessing==1.1.0
+Markdown==3.1.1
+numpy==1.17.1
+protobuf==3.9.1
+pyglet==1.3.2
+PyYAML==5.1.2
+-e git+https://github.com/HendrikPN/scigym.git@bf548ffb60e9c32f76af901cabdaf5a531d0a2ff#egg=scigym_pkg
+scipy==1.3.1
+six==1.12.0
+tensorboard==1.14.0
+tensorflow==1.14.0
+tensorflow-estimator==1.14.0
+termcolor==1.1.0
+Werkzeug==0.15.5
+wrapt==1.11.2
```

Comment:
Why do we have so many dependencies? In the `setup.py` it only mentions 2 extra dependencies. Aren't those the dependencies that should appear here?

Can you add comments that explain what environments these are needed for and which one are _general dependencies_?

Reply:
Some of the dependencies (lets call them parents) also have dependencies (children) - I think its good practice to list the children dependencies explicitly, as if the parent package hasn't listed the children versions specifically, then this could later cause problems when new children versions are released. 

I can make this file more structured, as you have suggested.

Reply:
Maybe we should remove this file (see also my answer to your comment below).

Reply:
see below...